src/tests/ui/planner/workitem-quickpreview.ts

```diff
@@ -110,6 +110,11 @@ export class WorkItemQuickPreview extends ui.BaseElement {
   effortTextArea = new ui.TextInput(this.$('[placeholder="Effort"]'), 'effort textarea');
   workItemsGroup = new ui.Clickable(this.element(by.cssContainingText('alm-dynamic-field .f8-dynamic-control', ' Work Items ')),'Side panel WorkItem button');
   businessValue = new ui.TextInput(this.$('textarea[placeholder="Business Value"]'), ' Business value textarea');
+  storyPoints = new ui.TextInput(this.$('textarea[placeholder="Storypoints"]'), ' Storypoints textarea');
+  dynamicFieldDiv = new ui.BaseElement(this.$('.dynamic-integer-field'), 'Dynamic Integer Field Div');
```

Comment:
@divyanshiGupta W/element - more than one element found for locator By(css selector, .dynamic-integer-field) - the first result will be used
test is failing because of this please update dynamicFieldDiv value